### ISSUE TITLE
Add field name to the error messages

### ISSUE THREAD

**Issue by arthur-kv** [hiddentao/squel#269]
Hello!

Sometimes it is very hard to detect invalid field value, especially in dynamically generated queries, so it would be very helpful to see a field name in such error messages:

```[Error: field value must be a string, number, boolean, null or one of the registered custom value types]```

It can be something like that:

``` Value of the field "field_name" must be a string, number, boolean, null or one of the registered custom value types] ```

Thanks!

**Comment by hiddentao** [comment]
Good point. This will require a bit of work across the codebase to get working right. Will add to next milestone.

**Comment by arthur-kv** [comment]
@hiddentao great! Thanks!

**Comment by FrancescoBorzi** [comment]
any plans to implement this?
